app/channels/api_channel.rb

```diff
@@ -126,4 +126,9 @@ def task_criterions
   def task_users
     User.joins(contest: :tasks).where(tasks: { id: task_id })
   end
+
+  def ready_info
+    task = Task.find task_id
+    { contest: task.contest.display_name, task: task.display_name }
```

Comment:
For future readability, `task_name` and `contest_name` would be better.

Reply:
ok